repo/repo.go

```diff
@@ -75,7 +75,7 @@ func UpdatedFiles(commit string) (files []string, err error) {
 }
 
 func Version(path string, dependencies []string, excluded []string) string {
-	args := []string{"log", "-1", "--format=%h", "--", path}
+	args := []string{"log", "-1", "--format=%H", "--", path}
```

Comment:
Again, you might want to use 12 chars rather than the whole SHA

Reply:
cf https://github.com/missena-corp/gally/pull/67/files#diff-cfcc49579773f33cf09615ee3dd7a70dR87